Capstone_Project/BaseLine.ipynb

```diff
@@ -1,5783 +0,0 @@
-       "      <td>Bar/Cocktail</td>\n",
-       "      <td>35.708146</td>\n",
-       "      <td>139.666288</td>\n",
-       "      <td>0</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>322335</th>\n",
-       "      <td>air_fff68b929994bfbd</td>\n",
-       "      <td>5</td>\n",
-       "      <td>16</td>\n",
-       "      <td>2017-04-21</td>\n",
-       "      <td>4</td>\n",
-       "      <td>Tōkyō-to Nakano-ku Nakano</td>\n",
-       "      <td>Bar/Cocktail</td>\n",
-       "      <td>35.708146</td>\n",
-       "      <td>139.666288</td>\n",
-       "      <td>0</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>322336</th>\n",
-       "      <td>air_fff68b929994bfbd</td>\n",
-       "      <td>5</td>\n",
-       "      <td>16</td>\n",
-       "      <td>2017-04-22</td>\n",
-       "      <td>5</td>\n",
-       "      <td>Tōkyō-to Nakano-ku Nakano</td>\n",
-       "      <td>Bar/Cocktail</td>\n",
-       "      <td>35.708146</td>\n",
-       "      <td>139.666288</td>\n",
-       "      <td>0</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n",
-       "</div>"
-      ],
-      "text/plain": [
-       "                air_store_id  reserve_visitors  time_diff visit_date  \\\n",
-       "322332  air_fff68b929994bfbd                 5         16 2017-04-18   \n",
-       "322333  air_fff68b929994bfbd                 5         16 2017-04-19   \n",
-       "322334  air_fff68b929994bfbd                 5         16 2017-04-20   \n",
-       "322335  air_fff68b929994bfbd                 5         16 2017-04-21   \n",
-       "322336  air_fff68b929994bfbd                 5         16 2017-04-22   \n",
-       "\n",
-       "        visitors              air_area_name air_genre_name   latitude  \\\n",
-       "322332         6  Tōkyō-to Nakano-ku Nakano   Bar/Cocktail  35.708146   \n",
-       "322333         2  Tōkyō-to Nakano-ku Nakano   Bar/Cocktail  35.708146   \n",
-       "322334         2  Tōkyō-to Nakano-ku Nakano   Bar/Cocktail  35.708146   \n",
-       "322335         4  Tōkyō-to Nakano-ku Nakano   Bar/Cocktail  35.708146   \n",
-       "322336         5  Tōkyō-to Nakano-ku Nakano   Bar/Cocktail  35.708146   \n",
-       "\n",
-       "         longitude  holiday_flg  \n",
-       "322332  139.666288            0  \n",
-       "322333  139.666288            0  \n",
-       "322334  139.666288            0  \n",
-       "322335  139.666288            0  \n",
-       "322336  139.666288            0  "
-      ]
-     },
-     "execution_count": 226,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "df_total.tail()"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 227,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# have a single column of visitors instead of two\n",
-    "df_total['visitors'] = df_total['visitors'] + df_total['reserve_visitors']"
```

Comment:
@nishalpattan :
Why adding visitors plus reserve visitors? Can you give me some context so that I can understand?

Reply:
I thought both would represent the single feature, like the total number of visitors.

Reply:
@nishalpattan : Wild guess here but would not "reserve visitors" be the initial number of customers who in principle that were going to show at the restaurant but at the end only the number under "visitors" show?